public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -231,17 +231,41 @@ export function useAllAlertmanagerAbilities(): Abilities<AlertmanagerAction> {
       // TODO: Move this into the permissions config and generalise that code to allow for an array of permissions
       isGrafanaFlavoredAlertmanager ? AccessControlAction.AlertingReceiversCreate : null
     ),
-    [AlertmanagerAction.ViewContactPoint]: toAbility(AlwaysSupported, notificationsPermissions.read),
-    [AlertmanagerAction.UpdateContactPoint]: toAbility(hasConfigurationAPI, notificationsPermissions.update),
-    [AlertmanagerAction.DeleteContactPoint]: toAbility(hasConfigurationAPI, notificationsPermissions.delete),
+    [AlertmanagerAction.ViewContactPoint]: toAbility(
+      AlwaysSupported,
```

Comment:
Not related to this PR , but I'm wondering why all the read actions has the `AlwaysSupported`, whereas the updates have the `hasConfigurationAPI` instead 🤔